transcom/mymove
# INT-B-20043

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A950357&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## Summary

Blocking SAFETY moves from office users without SAFETY privileges

## How to test
1. Create a safety move
2. Login as an office user with safety privileges
3. Copy the safety move url
4. Login as an office user without safety privileges
5. Try to navigate to safety move with url
6. Do not see the page
